alf/utils/dist_utils.py

```diff
@@ -468,6 +472,8 @@ def __init__(self, base_dist: td.Distribution, loc, scale):
             loc (Tensor or float): Location parameter.
             scale (Tensor or float): Scale parameter.
         """
+        loc = torch.as_tensor(loc)
+        scale = torch.as_tensor(scale)
```

Comment:
I think you need to add this to `AffineTransform` as well in case a scalar float is passed.

Reply:
> I think you need to add this to `AffineTransform` as well in case a scalar float is passed.

Right. Updated.